src/creature.cpp

```diff
@@ -909,16 +915,16 @@ void Creature::goToFollowCreature()
 
 			if (dir != DIRECTION_NONE) {
 				listWalkDir.clear();
-				listWalkDir.push_front(dir);
+				listWalkDir.emplace_back(dir);
```

Comment:
```suggestion
				listWalkDir.push_back(dir);
```

`emplace*` only makes sense when constructing in-place (i.e. passing constructor parameters).

Reply:
Fixed.